File: webapp/backend/src/main/java/com/computronica/webapp/model/Calificaciones.java
// src/main/java/com/computronica/webapp/model/Calificaciones.java
package com.computronica.webapp.model;

import com.google.cloud.Timestamp;

import lombok.Data;

@Data
public class Calificaciones {
    private String id;
    private String estudianteId;
    private String asignaturaId;
    private String evaluacion;
    private double nota;
    private Timestamp fecha;
}

File: CompuTronica/app/src/main/java/com/example/computronica/Model/Calificaciones.kt
package com.example.computronica.Model

import com.google.firebase.Timestamp

data class Calificaciones(
    var id: String = "",
    val estudianteId: String = "",
    val asignaturaId: String = "",
    val evaluacion: String = "",
    val nota: Double = 0.0,
    val fechaRegistro: String = ""
)

